tests/sentry/runner/commands/badstrict.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: strict-options
+data:
+  # the yaml options file that gets passed from k8 job to senty runner
+  options-strict.yaml:
+    github-login.base-domain : {"test" : 123}
```

Comment:
You can use standard yaml, no need to mix yaml with the json style map representation:
```
options-strict.yaml:
    github-login.base-domain:
       test: 123
```